### ISSUE TITLE
WhiteSource is called Mend

### ISSUE THREAD

**Issue by winterrocks** [OpenChain-Project/Reference-Material#55]
## **Replace WhiteSource with Mend**
https://github.com/OpenChain-Project/Reference-Material/blob/master/Adoption-Preparation/Model-Provisions/openchain-standards-model-provisions.0.5.md#q4do-you-use-any-software-tools-or-services-to-assist-with-open-source-license-compliance-for-example-black-duck-scancode-white-source-fossology-sw360

WhiteSource changed their name to Mend already some time ago and I'm suggesting that on this Q4 we'll use Mend instead of WhiteSource

**Additional context**
Let's use the current name. Of course we could list many other tools too, but it says "For example" so there is no need to list dozens of tools.

**Comment by andrewjskatz** [comment]
Good point. I'm not sure that the Mend.io has much name recognition yet, and this whole space is subject to constant changes of name and ownership, so I'm tempted to delete the reference entirely. Black Duck still seems to be called Black Duck, even within Synopsys. References to FOSSID/Snyk, Palamida/Revenera/Flexera would just be confusing as well, I think. I do want to keep at least one proprietary name in there, however, to ensure that there is no implication that we are only interested in open source tools as the answer to this question.

**Comment by myagi2019** [comment]
In my understanding this activity was closed and will not be taken further forward. Therefore I propose @winterrocks and @andrewjskatz that this issue is closed.

**Comment by winterrocks** [comment]
Closing. 
@andrewjskatz please reopen if you think there is a need to do so.
